src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -107,7 +108,7 @@ public GameData loadGame(final InputStream inputStream, final String savegamePat
             return null;
           }
           final boolean closeCurrentInstance = buttonPressed.equals(yesClose);
-          GameRunner.startGame(savegamePath, newClassPath, null);
```

Comment:
Avoid using Optional in parameters. Here we should overload `startGame` to only need two args instead of 3. When looking at a method arg being passed, there is not much difference between `null` and `Optional.empty()`, they both tell you very little.